src/templates/Mouf/Html/Widgets/JqueryFileUpload/JqueryFileUploadWidget.php

```diff
@@ -42,17 +42,17 @@
         var acceptFileTypes = <?= $acceptFileTypes ?>;
         if((data.originalFiles[0]['type'].length && !acceptFileTypes.test(data.originalFiles[0]['type']))
                 && (data.originalFiles[0]['name'].length && !acceptFileTypes.test(data.originalFiles[0]['name']))) {
-            uploadErrors.push('Not an accepted file type');
+            uploadErrors.push('<?= $acceptFileTypesMessage ?>');
         }
         <?php } ?>
         <?php if ($minFileSize) { ?>
-        if(data.originalFiles[0]['size'].length && data.originalFiles[0]['size'] < <?= $minFileSize ?>) {
-            uploadErrors.push('Filesize is too small');
+        if(data.originalFiles[0]['size'].toString().length && data.originalFiles[0]['size'] < <?= $minFileSize ?>) {
+            uploadErrors.push('<?= $minFileSizeMessage ?>');
         }
         <?php } ?>
         <?php if ($maxFileSize) { ?>
-        if(data.originalFiles[0]['size'].length && data.originalFiles[0]['size'] > <?= $maxFileSize ?>) {
-            uploadErrors.push('Filesize is too big');
+        if(data.originalFiles[0]['size'].toString().length && data.originalFiles[0]['size'] > <?= $maxFileSize ?>) {
+            uploadErrors.push('<?= $maxFileSizeMessage ?>');
```

Comment:
WARNING! There is a missing json_encode here!!!
If there is a quote in the message, there will be a bug in the Javascript!

You shoult instead write: 

```
uploadErrors.push(<?= json_encode($maxFileSizeMessage) ?>);
```

The same applies to the 2 insertions in Javascript code above!!!